src/System.Drawing.Common/src/System/Drawing/Printing/PreviewPrintController.Unix.cs

```diff
@@ -74,31 +65,26 @@ public override void OnEndPrint(PrintDocument document, PrintEventArgs e)
 
         public override Graphics OnStartPage(PrintDocument document, PrintPageEventArgs e)
         {
+            if (document == null)
+            {
+                throw new ArgumentNullException(nameof(document));
+            }
+            if (e == null)
+            {
+                throw new ArgumentNullException(nameof(e));
+            }
```

Comment:
Same question for all of these additions.  They don't seem like the right trade-off to me.